### DISCUSSION TITLE
I need to have 0.001 ETH in my mainnet

### DISCUSSION THREAD

**Discussion by Nenepo** [Cyfrin/foundry-full-course-cu#2872] [Q&A]
In the GitHub docs the faucet.chain.link doesn’t require eth in the mainnet, but I’m on it now and they do require eth in the mainnet

**Comment by EngrPips** [comment]
Hello @Nenepo, Getting sepolia eth is difficult lately, so I will advise that you test stuff locally or use another test net

**Comment by bushcode** [comment]
@Nenepo I was able to get some test net sepolia eth from here https://cloud.google.com/application/web3/faucet/ethereum/sepolia. Hope it works for you.

**Reply by EngrPips** [reply]
Thanks for the heads-up.

**Comment by NishantCoder108** [comment]
Thank you @bushcode , It is working for me .

**Reply by EngrPips** [reply]
Awesome.
